docs/reference.md

```diff
@@ -193,6 +193,9 @@ if (await isAuthorizedAsync()) {
 Used to start the checkout flow and collect payment information from Square
 Reader.
 
+**Notice**: You can not start checkout flow from a modal screen, close the modal
```

Comment:
Rather than making it a note, just add it to the paragraph above. For example:

Used to start the checkout flow and collect payment information from Square Reader. Checkout cannot be started from a modal screen.